workloads/tools/ueransim/Kptfile

```diff
@@ -5,8 +5,17 @@ metadata:
   annotations:
     config.kubernetes.io/local-config: "true"
 info:
-  description: this package represents ueransim NFs, which are required to perform E2E connectivity testing
+  description: this package represents ueransim NFs, which are required to perfirm E2E conn testing
 pipeline:
   mutators:
+    - image: gcr.io/kpt-fn/apply-replacements:v0.1.1
+      configPath: apply-replacements-owner.yaml
+    - image: gcr.io/kpt-fn/apply-replacements:v0.1.1
+      configPath: apply-replacements-namespace.yaml
     - image: gcr.io/kpt-fn/set-namespace:v0.4.1
       configPath: package-context.yaml
+    - image: docker.io/nephio/ueransim-deploy-fn:v1.0.1
```

Comment:
@electrocucaracha Here we can just use latest right for the moment and tag it it v2.0 when it is there or you want to use v1.0.1?